src/theory/booleans/proof_checker.cpp

```diff
@@ -25,6 +25,7 @@ void BoolProofRuleChecker::registerTo(ProofChecker* pc)
   pc->registerChecker(PfRule::SPLIT, this);
   pc->registerChecker(PfRule::RESOLUTION, this);
   pc->registerChecker(PfRule::CHAIN_RESOLUTION, this);
+  pc->registerChecker(PfRule::MACRO_RESOLUTION_TRUST, this);
```

Comment:
This should be a trusted rule with a pedantic level, e.g. 3, see
https://github.com/CVC4/CVC4/blob/master/src/theory/builtin/proof_checker.cpp#L73

Reply:
done